backend/src/graphql/resolver/ContributionResolver.test.ts

```diff
@@ -82,55 +93,117 @@ afterAll(async () => {
+    admin = await userFactory(testEnv, peterLustig)
+    await userFactory(testEnv, raeuberHotzenplotz)
+    const bibisCreation = creations.find((creation) => creation.email === 'bibi@bloxberg.de')
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    bibiCreatedContribution = await creationFactory(testEnv, bibisCreation!)
+    await mutate({
+      mutation: login,
+      variables: { email: 'bibi@bloxberg.de', password: 'Aa12345_' },
+    })
+    pendingContribution = await mutate({
+      mutation: createContribution,
+      variables: {
+        amount: 100.0,
+        memo: 'Test PENDING contribution',
+        creationDate: new Date().toString(),
+      },
+    })
+    inProgressContribution = await mutate({
+      mutation: createContribution,
+      variables: {
+        amount: 100.0,
+        memo: 'Test IN_PROGRESS contribution',
+        creationDate: new Date().toString(),
+      },
+    })
+    contributionToConfirm = await mutate({
+      mutation: createContribution,
+      variables: {
+        amount: 100.0,
+        memo: 'Test contribution to confirm',
+        creationDate: new Date().toString(),
+      },
+    })
+    contributionToDeny = await mutate({
+      mutation: createContribution,
+      variables: {
+        amount: 100.0,
+        memo: 'Test contribution to deny',
+        creationDate: new Date().toString(),
+      },
+    })
+    contributionToDelete = await mutate({
+      mutation: createContribution,
+      variables: {
+        amount: 100.0,
+        memo: 'Test contribution to delete',
+        creationDate: new Date().toString(),
+      },
+    })
+    await mutate({
+      mutation: login,
+      variables: { email: 'peter@lustig.de', password: 'Aa12345_' },
+    })
+    await mutate({
+      mutation: adminCreateContributionMessage,
+      variables: {
+        contributionId: inProgressContribution.data.createContribution.id,
+        message: 'Test message to IN_PROGRESS contribution',
+      },
+    })
+    await mutate({
+      mutation: logout,
+    })
+    resetToken()
+  })
+
+  afterAll(async () => {
+    await cleanDB()
+    resetToken()
+  })
+
   describe('createContribution', () => {
     describe('unauthenticated', () => {
       it('returns an error', async () => {
-        await expect(
-          mutate({
-            mutation: createContribution,
-            variables: { amount: 100.0, memo: 'Test Contribution', creationDate: 'not-valid' },
-          }),
-        ).resolves.toEqual(
-          expect.objectContaining({
-            errors: [new GraphQLError('401 Unauthorized')],
-          }),
-        )
+        const { errors: errorObjects }: { errors: [GraphQLError] } = await mutate({
+          mutation: createContribution,
+          variables: { amount: 100.0, memo: 'Test Contribution', creationDate: 'not-valid' },
+        })
+
+        expect(errorObjects).toMatchObject([new GraphQLError('401 Unauthorized')])
```

Comment:
In general:
```suggestion
        expect(errorObjects).toEqual([new GraphQLError('401 Unauthorized')])
```
We always want to use the strongest available check. To equal is deep equality of objects.

Reply:
Done for all Errors.